distribution/ddf-common/src/main/resources/bin/setenv

```diff
@@ -64,5 +64,18 @@ export EXTRA_JAVA_OPTS
 # export KARAF_DATA # Karaf data folder
 # export KARAF_BASE # Karaf base folder
 
-export KARAF_OPTS="-Dfile.encoding=UTF8 -Dddf.home=$DDF_HOME"
+#
+# Admins can uncomment the following line and comment out the other definition of
+# the KARAF_SYSTEM_OPTS variable in order to determine any missing security permissions
+# should they install third-party bundles requiring additional access.
```

Comment:
❓ If uncommenting this line grants everything by default, we might want to add a comment to that regard

Reply:
Good point. It is a risk if left on. I'll expand the comment here and in the windows script.